app/javascript/app/service-provider-form.js

```diff
@@ -10,17 +10,25 @@ $(function(){
 
   var toggle_form_fields = function(id_protocol){
     switch(id_protocol){
-      case 'openid_connect':
+      case 'openid_connect_private_key_jwt':
```

Comment:
I wanted to add/update specs for this but we currently don't have any 😞, or rather the specs that exist are all skipped. I'm questioning the utility of adding those specs since this whole site needs a major overhaul, but I could be convinced to try and write them.